### PR TITLE
Ckpt docs

### LINKED ISSUE #6: Instructions for using checkpoints
Model checkpoint instructions should be more explicit than the current "To skip model training, download pre-trained models." Where do the checkpoints go? What to do about the fact that the `.yaml` files for the models linked on quilt have different names from their copies in github?

### PR DESCRIPTION
**Pull request by pgarrison**
# Issue
We need to explain how to download and use the checkpoints.

Closes #6 

# Changes
* Write a table describing how to copy checkpoints to folders like `configs/experiment/cellpack/ckpts`.
* For completeness, replace the single example of `run_embeddings.py` with a table of all 6 options.
* Some other miscellaneous clean up, partially because of `pre-commit`

# Reviewing
* See [new docs here](https://github.com/AllenCell/benchmarking_representations/blob/ckpt-docs/docs/USAGE.md)
* @ritvikvasan Will the models actually read checkpoints from `configs/experiment/cellpack/ckpts` without further changes? I know that's where the checkpoints are output, but I don't know if they read from there too.
* If we like this thorough "table of commands" format I can do it for the benchmarking analysis steps too (separate PR).

### CODE REVIEW

**Comment by ritvikvasan** [comment]
maybe we can update the `to this folder` section in your table, with the corresponding result config?

File: docs/USAGE.md [hunk 1]
```diff
@@ -42,102 +46,124 @@ pip install -e .
...
+- [WTC-11 hIPSc single cell image dataset v1 polymorphic structures](https://open.quiltdata.com/b/allencell/tree/aics/morphology_appropriate_representation_learning/model_embeddings/other_polymorphic/)
+- [Nucleolar drug perturbation dataset](https://open.quiltdata.com/b/allencell/tree/aics/morphology_appropriate_representation_learning/model_embeddings/npm1_perturb/)
 
 ## Steps to run benchmarking analysis
 
 1. To compute benchmarking features from the embeddings and trained models, run
 
-```
+```bash
 python src/br/analysis/run_features.py --save_path "/outputs_cellpack/" --embeddings_path "./morphology_appropriate_representation_learning/model_embeddings/cellpack" --sdf False --dataset_name "cellpack" --debug False
```
Comment (ritvikvasan) [inline docs/USAGE.md:163]: we can make a similar matrix/table for both run_features and run_analysis

File: docs/USAGE.md [hunk 2]
```diff
@@ -42,102 +46,124 @@ pip install -e .
...
-```
-python src/br/analysis/run_embeddings.py --save_path "./outputs/" --sdf False --dataset_name "pcna" --batch_size 5 --debug False
-```
+## Steps to download pre-trained models
+
+To skip model training, download pre-trained models. For each of the six datasets, there are five `.ckpt` files to download.
+
+| Download `.ckpt` files from here                                                                                                                                                                         | To this folder                               |
+| -------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- | -------------------------------------------- |
+| [cellPACK synthetic dataset](https://open.quiltdata.com/b/allencell/tree/aics/morphology_appropriate_representation_learning/model_checkpoints/cellpack/)                                                | `configs/experiment/cellpack/ckpts`          |
```
Comment (ritvikvasan) [inline docs/USAGE.md:122]: The "To this folder" does not need to be so strict

**Review by ritvikvasan — CHANGES_REQUESTED** [review]

File: docs/USAGE.md (hunk 2 above)
Comment (pgarrison) [reply docs/USAGE.md]: Removed the table entirely: [f416574](https://github.com/AllenCell/benchmarking_representations/pull/61/commits/f416574e47f28ec38bfa09206abcdc04a6d426ca)

**Comment by pgarrison** [comment]
Updated with instructions to download to `morphology_appropriate_representation_learning/model_checkpoints`

**Review by ritvikvasan — APPROVED** [review]
LGTM!

**Review by fatwir — APPROVED** [review]
LGTM!

**Pull request merged**